install.php

```diff
@@ -51,11 +53,23 @@
     $_POST = array_merge($_POST, $_POST[$_POST['smtp_tab_selected']]);
 }
 
-//session_destroy();
-if (version_compare(phpversion(),'5.2.0') < 0) {
-	$msg = 'Minimum PHP version required is 5.2.0.  You are using PHP version  '. phpversion();
+/**
+ * Check php version
+ *
+ * If less than minimum we refuse to install.
+ */
+if (check_php_version() === -1) {
+	$msg = 'The recommended PHP version to install SuiteCRM is ';
```

Comment:
I really don't know. At this point there is no definition of "Installation Language". This gets executed really early in the installation process, before loading the language definition files, the Localization support and all that...